projectDocs/dev/developerGuide/developerGuide.md

```diff
@@ -504,6 +505,24 @@ As a built-in app module, wwahost can be imported from nvdaBuiltin.appModules.
 Another difference is how the app module class is defined.
 As wwahost app module provides necessary infrastructure for apps hosted inside, you just need to subclass the wwahost AppModule class.
 
+### Example 3: an app module for  an app employing Edge WebView2 (msedgewebview2.exe) {#example3}
+
+The following example is an app module employing Edge WebView2 runtime with browse mode disabled by default, using modern Outlook (olk.exe) as an example.
+
+    --- start ---
+    # msedgewebview2 example (modern Outlook/olk.py)
+    # Developer guide example 3
+    
+    import appModuleHandler
+    
+    class AppModule(appModuleHandler.AppModule):
+    
+    	disableBrowseModeByDefault: bool = True
+    
+    --- end ---
+
+Browse mode is disabled for this example because apps employing WebView2 display their interfaces as webpages. You can remove "disableBrowseModeByDefault" line if you would like to let users navigate the app using browse mode commands.

```

Comment:
```suggestion
Browse mode is disabled for this example because apps employing WebView2 display their interfaces as webpages.
You can remove the "disableBrowseModeByDefault" line if you would like to let users navigate the app using browse mode commands.
```